io.github.diegopvlk.Dosage.json

```diff
@@ -0,0 +1,63 @@
+{
+	"app-id": "io.github.diegopvlk.Dosage",
+	"runtime": "org.gnome.Platform",
+	"runtime-version": "45",
+	"sdk": "org.gnome.Sdk",
+	"command": "io.github.diegopvlk.Dosage",
+	"finish-args": [
+		"--share=network",
+		"--share=ipc",
+		"--socket=fallback-x11",
+		"--device=dri",
+		"--socket=wayland"
+	],
+	"cleanup": [
+		"/include",
+		"/lib/pkgconfig",
+		"/man",
+		"/share/doc",
+		"/share/gtk-doc",
+		"/share/man",
+		"/share/pkgconfig",
+		"*.la",
+		"*.a"
+	],
+	"modules": [
+		{
+			"name": "libportal",
+			"buildsystem": "meson",
+			"config-opts": ["-Ddocs=false", "-Dvapi=false", "-Dbackends=gtk4"],
+			"sources": [
+				{
+					"type": "archive",
+					"url": "https://github.com/flatpak/libportal/releases/download/0.6/libportal-0.6.tar.xz",
+					"sha256": "88a12c3ba71bc31acff7238c280de697d609cebc50830c3766776ec35abc6566"
+				}
+			]
+		},
+		{
+			"name": "blueprint-compiler",
+			"cleanup": ["*"],
+			"buildsystem": "meson",
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+					"tag": "v0.10.0"
+				}
+			]
+		},
+		{
+			"name": "dosage",
```

Comment:
In the appdata, can you pull the screenshots from the tag instead of `main`?

This will protect against missing screenshots in the future and the build here breaking.